vscode-wpilib/resources/webviews/projectcreator.html

```diff
@@ -17,39 +17,41 @@
 <body>
   <img src="replaceresource/resources/wpilib.svg" width="100" />
 
-  <h1>Welcome to WPILib New Project Creator</h1>
+  <h1 data-i18n-trans>Welcome to WPILib New Project Creator</h1>
 
-  <button id="projectType" >Select a project type (Example or Template)</button>
+  <button id="projectType" data-i18n-trans>Select a project type (Example or Template)</button>
 
-  <button id="language" disabled>Select a language</button>
+  <button id="language" data-i18n-trans disabled>Select a language</button>
 
-  <button id="projectBase" disabled>Select a project base</button>
+  <button id="projectBase" data-i18n-trans disabled>Select a project base</button>
   <br/>
-  <br/> Select a folder to place the new project into.
+  <br/> <span data-i18n-trans>Select a folder to place the new project into.</span>
   <br/>
   <input id="projectFolder" type="text" style="width: 100%; margin-top: 10px; margin-bottom: 10px" />
   <br/>
-  <button id="projectSelectButton" >Select a new project folder</button>
+  <button id="projectSelectButton" data-i18n-trans>Select a new project folder</button>
   <br/>
-  <br/> Create new folder? Highly recommended to be checked
+  <br/> <span data-i18n-trans>Create new folder?</span> <span data-i18n-trans>Highly recommended to be checked</span>
   <input id="newFolderCB" type="checkbox" checked/>
   <br/>
   <br/>
-  <div id="projectnamediv">Enter a project name</div>
+  <div id="projectnamediv" data-i18n-trans="ui">Enter a project name</div>
   <input id="projectName" type="text" style="width: 100%; margin-top: 10px; margin-bottom: 10px"
   />
   <br/>
-  <div id="teamnumberdiv">Enter a team number</div>
+  <div id="teamnumberdiv" data-i18n-trans="ui">Enter a team number</div>
   <input id="teamNumber" type="text" style="width: 100%; margin-top: 10px; margin-bottom: 10px"
   />
   <br/>
   <div id="desktopdiv">
-    This is needed for simulation and unit testing support, however there are some cases where this will do
-    some unexpected things during build. In addition, not all vendor libraries support desktop.
-    This option can be set with the command "WPILib: Set Desktop Support" at any time.
+	<span data-i18n-trans data-i18n-key="hintDesktopSupport">
+      This is needed for simulation and unit testing support, however there are some cases where this will do

```

Comment:
![image](https://user-images.githubusercontent.com/7727148/65819746-06360d00-e1d5-11e9-895c-3fd107f40788.png)

For some reason, this is just showing `hintDesktopSupport` rather then the actual text.